modules/installation-user-infra-exporting-common-variables-arm-templates.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-user-infra-exporting-common-variables-arm-templates_{context}"]
+= Exporting common variables for ARM templates
+
+You must export a common set of variables that are used with the provided Azure
+Resource Manager (ARM) templates used to assist in completing a user-provided
+infrastructure install on Microsoft Azure.
+
+[NOTE]
+====
+Specific ARM templates can also require additional exported variables, which are
+detailed in their related procedures.
+====
+
+.Prerequisites
+
+* Obtain the {product-title} installation program and the pull secret for your cluster.
+
+.Procedure
+
+. Export common variables found in the `install-config.yaml` to be used by the
+provided ARM templates:
++
+----
+$ export CLUSTER_NAME=<cluster_name><1>
+$ export AZURE_REGION=<azure_region><2>
+$ export SSH_KEY=<ssh_key><3>
+$ export BASE_DOMAIN=<base_domain><4>
+$ export BASE_DOMAIN_RESOURCE_GROUP=<base_domain_resource_group><5>
+----
+<1> The value of the `.metadata.name` attribute from the `install-config.yaml` file.
+<2> The region to deploy the cluster into, for example `us-east1`. This is the value of the `.platform.azure.region` attribute from the `install-config.yaml` file.
+<3> The SSH RSA public key file as a string. This is the value of the `.sshKey` attribute from the `install-config.yaml` file.
```

Comment:
I think we should add examples for the exports, but if there's only one to add it would be be this one because the string should contain spaces

Reply:
Sure; I can add examples for these exports. Here's an initial stab at an example based on what I observed:
```
$ export CLUSTER_NAME=test-cluster
$ export AZURE_REGION=us-east-1
$ export SSH_KEY=ssh-rsa xxx/xxx/xxx= user@email.com
$ export BASE_DOMAIN=example.com
$ export BASE_DOMAIN_RESOURCE_GROUP=ocp-cluster
```
I can also add to the SSH description that it should contain spaces. Please let me know any necessary edits to make important characteristics easier to spot.